charts/nri-bundle/values.yaml

```diff
@@ -15,6 +15,11 @@ kube-state-metrics:
   # This is mandatory if `infrastructure.enabled` is set to `true` and the user does not provide its own instance of KSM version >=1.8 and <=2.0
   enabled: false
 
+kube-state-metrics-v26:
+  # kube-state-metrics-v26.enabled -- Install the [`kube-state-metrics` chart](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) from the stable helm charts repository.
+  # This is mandatory if `infrastructure.enabled` is set to `true` and the user does not provide its own instance of KSM version >=1.8 and <=2.0
```

Comment:
As I see it, including some insight regarding when this or `kibe-state-metrics.enabled` should be used would be useful as it would be updated as well in the README chart docs.